src/hooks/vault/useVaults.ts

```diff
@@ -1,14 +1,16 @@
 import { useContractRead, useContractReads } from "wagmi";
 import registryContractJson from "../../abi/Registry.json";
-import { ethers } from "ethers";
+import { BigNumber, ethers } from "ethers";
 import { useEffect, useState } from "react";
 import api from "../../apis/Api";
 
+const registryContractAddress =
+  process.env.REACT_APP_REGISTRY_CONTRACT ||
+  "0xd90AE997C32EdE8feCe39694460543868Da0d0D1";
```

Comment:
we should throw an error instead of providing a non-descript address if the registry contract is not provided